src/api/request/request.node.ts

```diff
@@ -21,12 +21,19 @@ export type RequestOptions = {
   onProgress?: (event: any) => void
 }
 
-type BaseResponse = {
+export type RequestResponse = {
   data: string
   headers: Headers
   status?: number
 }
 
+export type FailedResponse = {
```

Comment:
You move `FailedResponse` to base request
Nice catch!!! Now I don't need duplicate this type in other methods 🙌🏻

Reply:
You are welcome : )